verify.js

```diff
@@ -0,0 +1,28 @@
+const db = require('.');
+
+const noDuplicates = Object.keys(db)
+  .filter(key => key !== 'schemas')
+  .map(list => {
+    let numberOfItems;
+    let numberOfUniques;
+    if (list == 'companies') {
```

Comment:
Use `===` for equality checking by value.

Reply:
![Uploading 95574709...]